scripts/exglobal_stage_ic.sh

```diff
@@ -96,6 +192,21 @@ if [[ "${DO_WAVE:-}" = "YES" ]]; then
     (( rc != 0 )) && error_message "${source}" "${target}" "${rc}"
     err=$((err + rc))
   done
+elif [[ "${DO_WAVE:-}" = "YES" && "${selection}" = "gefs" ]]; then
+  member_dir=""
+  YMD=${gPDY} HH=${gcyc} generate_com -r COM_WAVE_RESTART
+  [[ ! -d "${COM_WAVE_RESTART}" ]] && mkdir -p "${COM_WAVE_RESTART}"
+    for member_dir in $(seq -w 0 $((NMEM_ENS - 1))); do
+    source="${BASE_CPLIC}/${CPL_ATMIC}/${YMD}${HH}/${member_dir}/ice/${PDY}.${cyc}0000.restart.gwes_30m"
```

Comment:
gwes_30m is not a constant and depends on the wave grid being used.
```suggestion
    source="${BASE_CPLIC}/${CPL_ATMIC}/${YMD}${HH}/${member_dir}/ice/${PDY}.${cyc}0000.restart.${waveGRD}"
```